PlayCover/ViewModel/AppsVM.swift

```diff
@@ -16,10 +16,12 @@ class AppsVM: ObservableObject {
     }
 
     @Published var apps: [PlayApp] = []
+    @Published var unfilteredApps: [PlayApp] = []
```

Comment:
The naming convention should match that of `StoreVM` so `unfilteredApps` = `apps`, and `apps` = `filteredApps`